app/models/dimensions/edition.rb

```diff
@@ -21,7 +21,7 @@ class Dimensions::Edition < ApplicationRecord
     latest_by_content_id(content_id, locale)
       .where.not(base_path: exclude_paths)
   end
-
+  scope :live_content, -> { where.not(document_type: %w[redirect gone]) }
```

Comment:
Maybe keep the newline separating the scopes from the methods?

Reply:
Also, I'm a bit hesitant about "live" content, mainly because the Publishing API uses the term "live" to talk about the "draft" and "live" content stores. Maybe "real" could be used instead?

Reply:
You have a point Chris; what I like a bout `live` is that it is use by the product owner, and people around him (non tech people).

When it comes to naming it `real`... not sure about this name to be honest. I would like to explore other names if possible?

What about `editable_content`? this would represent content that is of interest for content designers because they can act on it?